sphinxcontrib/datatemplates/directive.py

```diff
@@ -119,10 +119,18 @@ def _load_data(self, resolved_path):
 
 
 class DataTemplateYAML(DataTemplateWithEncoding):
+    option_spec = dict(DataTemplateBase.option_spec, **{
+        'multiple-documents': rst.directives.flag,
+    })
+
     def _load_data(self, resolved_path):
         with open(resolved_path,
                   'r',
                   encoding=self.options.get('encoding', 'utf-8-sig')) as f:
+            if 'multiple-documents' in self.options:
+                return tuple(
```

Comment:
I don't think a tuple is really appropriate here. A list feels more correct, since it is a sequence of items.

Reply:
changed in d9b0601